services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusService.kt

```diff
@@ -246,7 +246,8 @@ class HakemusService(
             hakemus.hakemusEntityData =
                 hakemusEntityData.copy(
                     cableReports =
-                        cableReports.plus(sentJohtoselvityshakemus.applicationIdentifier!!)
+                        cableReports.plus(sentJohtoselvityshakemus.applicationIdentifier!!),
+                    cableReportDone = true,
```

Comment:
Should this happen 5 lines later, after the application has been sent? The difference would be that whether the user ordered a new johtoselvitys at the same time would be stored in the form data PDF sent to Allu.

Reply:
Yesn you're right again! Thank you for noticing this. I moved it to its own function after the send.